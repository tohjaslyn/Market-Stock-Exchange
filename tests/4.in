ADD trader1 100.55
IMPORT trader1 APL 12
BALANCE trader1
INVENTORY trader1
AMOUNT trader1 APL
SELL trader1 APL 10 1.56
BOOK SELL
ORDER 0000
CANCEL SELL 0000
BUY trader1 NFL 10 1
BOOK BUY
EXIT
