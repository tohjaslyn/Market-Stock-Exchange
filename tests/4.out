$ Success.
$ Trader now has 12.00 units of APL.
$ $100.55
$ APL
$ 12.00
$ No trades could be made, order added to sell book.
$ 0000: SELL 10.00xAPL @ $1.56
$ 0000: SELL 10.00xAPL @ $1.56
$ Order successfully cancelled.
$ No trades could be made, order added to buy book.
$ 0001: BUY 10.00xNFL @ $1.00
$ Have a nice day.
